CHANGELOG.md

```diff
@@ -34,7 +34,7 @@ Read more about the session argument for computed fields in the [docs](https://h
 
 ### Bug fixes and improvements
 
-(Add entries here in the order of: server, console, cli, docs, others)
+- server: compile with GHC 8.10.1, closing a space leak with subscriptions. (close #4517) (#3388)
```

Comment:
I propose to leave the instructions in place, and not to have a newline between the `server` entry and the `console` entry.